support-frontend/app/controllers/Application.scala

```diff
@@ -138,7 +134,7 @@ class Application(
       paymentApiStripeEndpoint = paymentAPIService.stripeExecutePaymentEndpoint,
       paymentApiPayPalEndpoint = paymentAPIService.payPalCreatePaymentEndpoint,
       idUser = idUser,
-      preload = preload
+      preload = Nil
```

Comment:
if this isn't used, then remove it completely (it was only added for this call)
also remove the Preload case class and anywhere else it's used.

Reply:
👍